.github/workflows/ci.yml

```diff
@@ -14,27 +14,26 @@ env:
 jobs:
   linting:
     runs-on: ubuntu-latest
-
+    env:
+      BUNDLE_WITHOUT: 'development test'
+      BUNDLE_WITH: 'lint'
     steps:
-      - uses: actions/checkout@v3
-
+      - uses: actions/checkout@v4
       - name: Set up Ruby 2.6
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 2.6
+          bundler-cache: true # 'bundle install' and cache gems
```

Comment:
Oh this is very clever!
This way we don't need to guess the right `bundler` version, love it!